docs/Using-Fleet/Mobile-device-management.md

```diff
@@ -14,19 +14,39 @@ _Available in Fleet Premium_
 
 End users can be reminded and encouraged to update macOS (via [Nudge](https://github.com/macadmins/nudge)).
 
-When a minimum version and deadline is saved in Fleet, the end user sees the below window until their macOS version is at or above the minimum version.
+When a minimum version and deadline is saved in Fleet, the end user sees the below Nudge window until their macOS version is at or above the minimum version. 
 
 To set the macOS updates settings in the UI, visit the **Controls** section and then select the **macOS updates** tab. To set the macOS updates settings programmatically, use the configurations listed [here](https://fleetdm.com/docs/using-fleet/configuration-files#mdm-macos-updates).
 
-![Fleet's architecture diagram](https://raw.githubusercontent.com/fleetdm/fleet/main/docs/images/nudge-window.png)
+![Nudge window](https://raw.githubusercontent.com/fleetdm/fleet/main/docs/images/nudge-window.png)
 
 As the deadline gets closer, Fleet provides stronger encouragement.
 
 If the end user has more than 1 day until the deadline, the Nudge window is shown everyday. The end user can defer the update and close the window.
 
 If there is less than 1 day, the window is shown every 2 hours. The end user can defer and close the window.
 
-If the end user is past the deadline, Fleet shows the window and end user can't close the window until they upgrade.
+If the end user is past the deadline, Fleet shows the window and end user can't close the window until they update.
+
+### End user experience
+
+Fleet automatically downloads macOS updates for the end user so that they don't have to. This way, end users don't have to wait for the update to download before they can install it.
+
+> Fleet automatically downloads updates on Intel Macs. On Macs with Apple silicon (e.g. M1), end users may have to download the update before they install it. Apple doesn't support downloading the update for the end user on Macs with Apple silicon.
```

Comment:
Seems repetitive that the last sentence says "Fleet automatically downloads" but then the next sentence says "it only happens on Intel macs." Could these be combined? 

> On macs with Intel chips, Fleet automatically ... . On macs with Apple Silicon, end users must...